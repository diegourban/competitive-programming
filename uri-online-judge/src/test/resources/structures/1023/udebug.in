1
10 156
1
10 179
1
10 188
3
1 10
2 20
3 30
3
1 7
2 7
3 7
0