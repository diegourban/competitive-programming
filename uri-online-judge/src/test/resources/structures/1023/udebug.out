Cidade# 1:
10-15
Consumo medio: 15.60 m3.

Cidade# 2:
10-17
Consumo medio: 17.90 m3.

Cidade# 3:
10-18
Consumo medio: 18.80 m3.

Cidade# 4:
6-10
Consumo medio: 10.00 m3.

Cidade# 5:
3-2 2-3 1-7
Consumo medio: 3.50 m3.
